dts/bindings/sensor/hamamatsu,s11059.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2022, Hiroki Tada
+# SPDX-License-Identifier: Apache-2.0
+
+description: HAMAMATSU PHOTONICS s11059 color sensor.
+
+compatible: "hamamatsu,s11059"
+
+include: i2c-device.yaml
+
+properties:
+
+  gain:
+    type: int
+    default: 0
```

Comment:
I think it's fine to keep `default` here, but the value needs to be justified in the description. Is it a power on reset register value?

Reply:
The low gain is the initial setting for this sensor, as described in the datasheet (bottom of page 3).
Since this property type has been changed to boolean, it is not necessary to set this `default`.